### DISCUSSION TITLE
Docs Contributing/Translations about Ellipses

### DISCUSSION THREAD

**Discussion by real-yfprojects** [borgbase/vorta#1164] [Translations]
On the [doc webpage](https://vorta.borgbase.com/contributing/translations/) it says that three dots should be used instead of an ellipses while tasks are in progress.
On the other hand the [KDE HIG](https://develop.kde.org/hig/style/writing/labels/#using-ellipses-in-labels) states:
> Ellipses are used to indicate that a button or menu item will perform an action that always requires additional user input before completing. Use the ellipsis Unicode character, not three dot characters, to let screen readers describe it appropriately. 
I would apply this to progress messages too.

What do you think?

**Comment by m3nu** [comment]
Maybe there was an issue with ellipses at the time of writing, which may or may not apply today. We can remove this note, but I wouldn't remove it in source code because all translations in Transifex would need to be updated too.

**Comment by bastiencyr** [comment]
I can add this rule to #1050 if needed.  I didnt find this consideration in other hig but it makes sense.

**Reply by m3nu** [reply]
Sure, so we don't forget it. Thanks!
